code_samples/user_management/oauth_google/config/packages/knpu_oauth2_client.yaml

```diff
@@ -0,0 +1,11 @@
+knpu_oauth2_client:
+    clients:
+        # configure your clients as described here: https://github.com/knpuniversity/oauth2-client-bundle#configuration
+        google:
+            type: google
+           client_id: '%env(OAUTH_GOOGLE_CLIENT_ID)%'
```

Comment:
These two `client_*` lines have the wrong indentation that will cause an exception.